Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
Image I've two entities, `Engine` and `Characteristic`, the first one has a name, a type and a one-to-one relation with  `Characteristic`, the second one has one column per characteristic.
If I build a form to edit an engine, I can edit the characteristic's fields with a key like `characteristic.puissance`. 
Do you see what I mean ?
 I will mask some characteristic depending on the selected type.